### PR TITLE
[@babel/code-frame_v7.x.x] Add definition

### PR DESCRIPTION
**Pull request by goodmind**
- Links to documentation: https://babeljs.io/docs/en/next/babel-code-frame.html
- Link to GitHub or NPM: npmjs.com/@babel/code-frame
- Type of contribution: new definition

### CODE REVIEW

File: definitions/npm/@babel/code-frame_v7.x.x/flow_v0.30.x-/code-frame_v7.x.x.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  declare export type SourceLocation = {
+    start: {|
+      line: number,
+      column?: number,
+    |},
+    end?: {|
+      line: number,
+      column?: number,
+    |},
+    ...
```
Comment (pascalduez) [inline definitions/npm/@babel/code-frame_v7.x.x/flow_v0.30.x-/code-frame_v7.x.x.js]: I guess we should scope this to an higher flow version, `flow_v0.84.x-` if I'm not mistaken.

**Comment by pascalduez** [comment]
@goodmind Thanks! Could you add a few tests?

File: definitions/npm/@babel/code-frame_v7.x.x/flow_v0.30.x-/code-frame_v7.x.x.js (hunk 1 above)
Comment (goodmind) [reply definitions/npm/@babel/code-frame_v7.x.x/flow_v0.30.x-/code-frame_v7.x.x.js]: Done

**Comment by pascalduez** [comment]
Dang, I get the same failing tests (plus others) while trying to work on the cli. Not related to this PR though.

**Pull request merged**